src/diffusers/pipelines/stable_video_diffusion/pipeline_stable_video_diffusion.py

```diff
@@ -161,7 +161,8 @@ def _encode_vae_image(
         num_videos_per_prompt,
         do_classifier_free_guidance,
     ):
-        image = image.to(device=device)
+        if device is not None:
+            image = image.to(device=device)
```

Comment:
```suggestion
        if device is not None:
            image = image.to(device=device)
```
I don't think this change is necessary no? `image.to(device=None)` is just a no-op and should be fine to write